ui/app/src/Alerts/Types.elm

```diff
@@ -18,17 +15,7 @@ type alias Alert =
     }
 
 
-type alias AlertGroup =
```

Comment:
Thanks for cleaning this up. This is a left over from switching to the /alerts endpoint. My bad.